rsconf/component/mongod.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""create nginx configuration
+
+:copyright: Copyright (c) 2017 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+from rsconf import component
+
+_DATA_DIR = "db"
+
+_MONGO_CONF_F = "/etc/mongod.conf"
+
+
+class T(component.T):
+    def internal_build_compile(self):
+        self.buildt.require_component("base_all")
+        self.j2_ctx, _ = self.j2_ctx_init()
```

Comment:
This should have the definitions and a unit_prepare, which returns run_d which is saved in j2_ctx in that call. This will call compt.service_prepare, which should be before the enable. run_u, data_d, should be set here.

Reply:
6c04a19